drivers/include/mtd_spi_nor.h

```diff
@@ -36,6 +36,24 @@ extern "C"
 {
 #endif
 
+#define STATUS_WIP  0x01u
+#define STATUS_WEL  0x02u
+#define STATUS_BP0  0x04u
+#define STATUS_BP1  0x08u
+#define STATUS_BP2  0x10u
+#define STATUS_BP3  0x20u
+#define STATUS_QE   0x40u
+#define STATUS_SRWD 0x80u
+
+#define SECURITY_SOTP 0x01u
+#define SECURITY_LDSO 0x02u
+#define SECURITY_PSB  0x04u
+#define SECURITY_ESB  0x08u
+#define SECURITY_XXXXX 0x10u
+#define SECURITY_PFAIL 0x20u
+#define SECURITY_EFAIL 0x40u
+#define SECURITY_WPSEL 0x80u
```

Comment:
Just move those to the `.c` file, no need to define them globally - then you can also get away without a prefix 😉